src/js/views/admin/program/workflows/workflows.scss

```diff
@@ -43,21 +43,21 @@
   text-align: center;
 }
 
-.workflows__action-ts {
+.workflows__timestamp {
   float: right;
   font-size: 12px;
   margin: 8px 0 0 8px;
   min-width: 40px;
   vertical-align: middle;
 }
 
-.workflows__action-icon {
+.workflows__icon {
   position: relative;
   top: -1px;
   vertical-align: 1px;
 
   .icon {
-    color: color(blue);
+    color: $black-60;
```

Comment:
The action icon is blue.  Only the flow folder is grey
https://marvelapp.com/68ec5id/screen/63607908